### PR TITLE
Rust 1.2 announcement

### PR DESCRIPTION
**Pull request by aturon**
r? @brson @steveklabnik

### CODE REVIEW

**Comment by steveklabnik** [comment]
LGTM from this Dr's office.

File: _posts/2015-08-06-Rust-1.2.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  [improvements to zero filling](https://github.com/rust-lang/rust/pull/26849)
+  that speed up `Vec::resize` and `Read::read_to_end`
+- The implementation of `Read::read_to_end` has been
+  [specialized for `stdin` and `File`](https://github.com/rust-lang/rust/pull/26950),
+  resulting in additional speedups.
+- The `PartialEq` implementation on slices is now
+  [much faster](https://github.com/rust-lang/rust/pull/26884).
+
+We have also made strides in our Windows support, landing
+[preliminary support for Windows XP](https://github.com/rust-lang/rust/pull/26601). While
```
Comment (nagisa) [inline _posts/2015-08-06-Rust-1.2.md]: s/support for/support for targeting/.

We still do not and probably never will support running rustc on XP.

**Comment by aturon** [comment]
@nagisa Fixed, thanks!

File: _posts/2015-08-06-Rust-1.2.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: "Announcing Rust 1.2"
+author: The Rust Core Team
+---
+
+Today marks the end of another Rust release cycle: we are [releasing][install]
```
Comment (huonw) [inline _posts/2015-08-06-Rust-1.2.md]: This sentence seems a bit awkward/doesn't suck me in (e.g. saying "release" twice, and the "end" seems negative).

File: _posts/2015-08-06-Rust-1.2.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+- The `PartialEq` implementation on slices is now
+  [much faster](https://github.com/rust-lang/rust/pull/26884).
+
+We have also made strides in our Windows support, landing
+[preliminary support for targeting Windows XP](https://github.com/rust-lang/rust/pull/26601). While
+we do not intend to treat Windows XP as a "first tier" platform, it is now
+feasible to build Rust code for XP as long as you avoid certain parts of the
+standard library.
+
+On the Cargo front, we have landed support for
```
Comment (huonw) [inline _posts/2015-08-06-Rust-1.2.md]: Just double checking, this was backported to 1.2, right?

Comment (huonw) [inline _posts/2015-08-06-Rust-1.2.md]: Oh... wait, this is the "what's in beta" section. Nevermind.

**Comment by brson** [comment]
lgtm

**Comment by marcusklaas** [comment]
:tada:

**Pull request merged**